### PR TITLE
feat : Space between links 

### LINKED ISSUE #9406: [BUG] No Space between add link and first link
### Description

On the [Links page](https://www.biodrop.io/account/manage/links) page, The Edit button of the first link seems to touch the `Add Link` button. We just need a little padding between the div containing the `Add Link`, `REORDER` buttons and the first link created by the user. Refer to the screenshot for a better understanding :)

### Screenshots

image

### Do you want to work on this issue?

None

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Vijaykv5**
## Fixes Issue 

closes #9406

## Changes proposed

On links page, Added a margin left on the heading of links.
Resized the Edit button to make it better look, Also changed the Alignment of the edit button by adding Respective padding between the divs.

**Before:**
Screenshot 2023-10-23 at 7 17 00 PM

**After**
Screenshot 2023-10-23 at 7 09 59 PM

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Vijaykv5** [comment]
@sital002 Please review this PR

File: components/account/manage/Edit.js [hunk 1]
```diff
@@ -2,15 +2,15 @@ import PencilIcon from "@heroicons/react/24/outline/PencilIcon";
 
 export default function Edit({ href, label = "", children }) {
   return (
-    <div className="relative w-full">
+    <div className="relative w-full  ">
```
Comment (akash19coder) [inline components/account/manage/Edit.js:5]: I have notices formatting issues in couple of places. Format your document (Shift + ALT + F) in VS Code.

Comment (Vijaykv5) [reply components/account/manage/Edit.js]: I formatted the files in new commit. Please check

File: .env.docker [hunk 2]
```diff
@@ -14,4 +14,4 @@ ADMIN_USERS="eddiejaoude,SaraJaoude"
 STRIPE_SECRET_KEY=""
 STRIPE_PREMIUM_PRICING_ID=""
 STRIPE_WEBHOOK_SECRET=""
-NEXT_PUBLIC_PREMIUM_SUPPORT_URL=""
+NEXT_PUBLIC_PREMIUM_SUPPORT_URL=""
```
Comment (eddiejaoude) [inline .env.docker:17]: Not sure what this change is for? Also GitHub marks it as red, please remove these changes

File: components/user/UserLink.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ export default function UserLink({
         animations[link.animation] !== animations.glow &&
           "dark:hover:bg-secondary-low/40 hover:bg-secondary-low/40",
         isEnabled && getLinkAnimation.get(animations[link.animation]),
-        "relative rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium grow",
+        "relative rounded-full border border-primary-medium-low dark:border-primary-medium-low dark:hover:border-[color:var(--hover-color)] hover:border-[color:var(--hover-color)] hover:shadow-xl pl-16 p-4 my-2 w-full content-start flex flex-row gap-4 items-center dark:bg-primary-medium grow",
```
Comment (eddiejaoude) [inline components/user/UserLink.js:49]: How does this affect other places the project uses the component `UserLink`?

**Comment by Vijaykv5** [comment]
> Thank you for the contribution 👍
> 
> But the changes have not be checked/tested properly. The update to the component breaks other areas
> 
> Screenshot 2023-10-25 at 09 12 04

It is fine right? Does it break anything?
Screenshot 2023-10-25 at 9 17 13 PM

**Comment by eddiejaoude** [comment]
It changes the design. I will speak with the other maintainers

**Comment by Vijaykv5** [comment]
> It changes the design. I will speak with the other maintainers

Cool Thanks.

**Comment by Vijaykv5** [comment]
Any updates?

**Comment by eddiejaoude** [comment]
It doesn't look good on mobile, please only do the issue task and not try to have scope creep - this causes more work and confusion for everyone

Screenshot 2023-10-28 at 08 15 54

**Comment by ChinmayMhatre** [comment]
@Vijaykv5  Hi , just wanted to checkin on your progress with the changes asked or if you needed any help!

**Comment by Vijaykv5** [comment]
@ChinmayMhatre Thank you for lending me the help!
May you can help me how can I fix that in mobile version. I tried many but its looking good for desktop, when applying to mobile as @eddiejaoude suggested it broken since same component is re-used to smaller view

**Comment by eddiejaoude** [comment]
@Vijaykv5 as I mentioned before, this PR has had scope creep, please just focus on the issue, so we can get it merged

**Comment by Abhishek-90** [comment]
@Vijaykv5 why did the edit button shifted to center when only adding `margin-top` will appropriate space between the `action button div` and `links div`?

File: components/user/UserLink.js (hunk 3 above)
Comment (eddiejaoude) [reply components/user/UserLink.js:49]: This breaks, especially for mobile view - this change is not needed for this issue

**Comment by eddiejaoude** [comment]
Closing PR and unassigning issue due to lack of activity for 2 weeks.

**Pull request closed without merging**